src/mcp_agent/workflows/llm/multipart_converter_azure.py

```
@@ -0,0 +1,361 @@
+    @staticmethod
+    def _create_fallback_text(
+        message: str, resource: Union[TextContent, ImageContent, EmbeddedResource]
+    ) -> TextContentItem:
+        if isinstance(resource, EmbeddedResource) and hasattr(resource.resource, "uri"):
+            uri = resource.resource.uri
+            return TextContentItem(text=f"[{message}: {getattr(uri, '_url', '')}]")
+        return TextContentItem(text=f"[{message}]")
+
+    @staticmethod
+    def convert_tool_result_to_azure(
+        tool_result: CallToolResult, tool_use_id: str
```

Pylint (3.3.7)

[warning] Unused argument 'tool_use_id'

(W0613)